mantle/kola/harness.go

```diff
@@ -884,36 +878,13 @@ func RegisterExternalTests(dir string) error {
 // getClusterSemVer returns the CoreOS semantic version via starting a
 // machine and checking
 func getClusterSemver(flight platform.Flight, outputDir string) (*semver.Version, error) {
```

Comment:
We've never implemented FCOS or RHCOS support in this function, and we don't seem likely to start writing native kola tests that use `MinVersion`/`EndVersion` to condition on the version of the guest.  So I'd be inclined to delete this function, tear out the supporting logic in the caller, and drop the `MinVersion`/`EndVersion` fields.